invoice/src/test/java/org/killbill/billing/invoice/api/user/TestInvoiceFlagBehaviors.java

```diff
@@ -85,7 +86,8 @@ public void testWrittenOffInvoiceBeforeAccountCredit() throws Exception {
         assertEquals(accountCBA1.compareTo(BigDecimal.ZERO), 0);
 
         // Add credit on the account
-        invoiceUserApi.insertCredit(accountId, BigDecimal.TEN, null, accountCurrency, true, null, null, null, callContext);
+        final InvoiceItem inputCredit = new CreditAdjInvoiceItem(null, accountId, clock.getUTCToday(), "some description", BigDecimal.TEN, accountCurrency, null);
```

Comment:
Maybe add an assertion in one of these tests that we can correctly retrieve the description.